.changeset/violet-wolves-hide.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Is the changeset necessary for this change?

Reply:
Yup it's for the `check_pr_has_changeset` check to pass, which is required.